.github/CODEOWNERS

```
@@ -664,6 +664,8 @@ peps/pep-0782.rst  @vstinner
 peps/pep-0783.rst  @hoodmane @ambv
 peps/pep-0784.rst  @gpshead
 # ...
+peps/pep-0787.rst  @ncoghlan @nhumrich
```

Unfortunately only people with write access to the repo can be added here
```suggestion
peps/pep-0787.rst  @ncoghlan```